actors/migration/nv15/top.go

```diff
@@ -57,6 +53,20 @@ type MigrationCache interface {
 	Load(key string, loadFunc func() (cid.Cid, error)) (cid.Cid, error)
 }
 
+func CidKey(c cid.Cid) string {
```

Comment:
I'm just using one catch-all key for all migrated objects (unlike the ActorHeadKey etc we had before). Is that bad?

Reply:
Ah nice missed this call out.  Its probably fine but better to be safe.  More details in [this review comment](https://github.com/filecoin-project/specs-actors/pull/1548#discussion_r777055395)

Reply:
Ack, will do!